sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items-batch-scoring/scripts/batch_scoring.py

```diff
@@ -2,17 +2,20 @@
 # Licensed under the MIT license.
 
 import os
-import argparse
-import json
 
-from azureml.core.model import Model
 from azureml.automl.core.shared import logging_utilities
 
 from azureml.automl.dnn.vision.common.logging_utils import get_logger
 from azureml.automl.dnn.vision.common.model_export_utils import (
     load_model,
     run_inference_batch,
 )
+
+# Uncomment line 16-18 and comment line 19-21 to run batch scoring
```

Comment:
nit: remove references to specific lines. If we add/remove imports we will have to update comment

Reply:
yaa.. I thought about this too... but it didn't very clear otherwise... what to uncomment, etc... 
anyways this file should not change very frequently...